FordLabs/PeopleMover
# Edit menu for product card will open and close correctly

Co-authored-by: Alex Wojtala <alexwojtala@gmail.com>
Co-authored-by: Michael Rygiel <michael.rygiel1@gmail.com>

## Issue
Resolves #490 

## What was done
- [x] Edit menu for product cards now opens and closes as expected

## How to test
1. Go to a space, and click on the edit menu of multiple products. Only one edit menu should be open at a time

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
